indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -0,0 +1,46 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_track_list(event) %}
+    {% if not event.tracks %}
+        {% call message_box('info', classes='no-rules-placeholder') %}
+            {% trans %}No tracks have been added yet.{% endtrans%}
+        {% endcall %}
+    {% else %}
+        <ul id="track-list" class="i-box-group vert" data-url="{{ url_for('.sort_tracks', event) }}">
```

Comment:
I don't really understand, what gets duplicated? And when I take it out of the macro: when you delete a track we only update `#track-list` not the whole page, and then the JS which applies the gradient doesn't work.